beacon_node/beacon_chain/src/merge_readiness.rs

```diff
@@ -145,18 +152,20 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             if let Err(e) = el.exchange_transition_configuration(&self.spec).await {
                 // The EL was either unreachable, responded with an error or has a different
                 // configuration.
-                return MergeReadiness::ExchangeTransitionConfigurationFailed(e);
+                return MergeReadiness::ExchangeTransitionConfigurationFailed {
+                    error: format!(
+                        "Error on exchange_transition_configuration endpoint {:?}",
```

Comment:
Can we please remove this string and just have the debug formatting of `e`? That'll keep the logs tidy :pray: